### PR TITLE
add warning for `DFTRACER_DATA_DIR`

### PR DESCRIPTION
**Pull request by rayandrew**
- added warning in `example.rst`
- added warning in `api.rst`

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -63,6 +63,10 @@ ENV Variables supported
    DFTRACER_LOG_FILE                STRING  PATH To log file. In this case process id and app name is appended to file.
    DFTRACER_DATA_DIR                STRING  Colon separated paths that will be traced for I/O accesses by profiler.
                                             For tracing all directories use the string "all" (not recommended).
+                                            Note: DFTRACER_DATA_DIR acts as a prefix. If both ``/local/scratch`` and
```
Comment (hariharan-devarajan) [inline docs/api.rst:66]:
```suggestion
                                            Note: DFTRACER_DATA_DIR uses a prefix tree. If both ``/local/scratch`` and
```

File: docs/examples.rst [hunk 2]
```diff
@@ -70,6 +70,12 @@ Example of running this configurations are:
     # Enable profiler
     DFTRACER_ENABLE=1
 
+.. warning::
+
+    Note: DFTRACER_DATA_DIR acts as a prefix. If both ``/local/scratch`` and
```
Comment (hariharan-devarajan) [inline docs/examples.rst:75]:
```suggestion
    Note: DFTRACER_DATA_DIR uses a prefix tree. If both ``/local/scratch`` and
```

**Review by hariharan-devarajan — CHANGES_REQUESTED** [review]
Small fix in language.

**Comment by rayandrew** [comment]
done, let me know if this looks good to you

**Review by hariharan-devarajan — APPROVED** [review]
Look good thanks

**Pull request merged**